### DISCUSSION TITLE
Does this work with react native?

### DISCUSSION THREAD

**Discussion by Seanmclem** [data-driven-forms/react-forms#1196] [General]
Does this work with React Native? Or, is there a similar equivalent library already for react native?

**Comment by rvsia** [comment]
@Seanmclem We have no experience with React Native so we cannot really respond to this question.

However, in the renderer package, there is at least one `div` used, so I suppose it won't work.

We are open to any changes to make it work.

**Comment by Seanmclem** [comment]
> @Seanmclem We have no experience with React Native so we cannot really respond to this question.
> 
> 
> 
> However, in the renderer package, there is at least one `div` used, so I suppose it won't work.
> 
> 
> 
> We are open to any changes to make it work.

OK, if I find the time soon I think I'll make a feature branch/fork to test and begin integration with React Native.
